.github/workflows/ci_workflows.yml

```diff
@@ -108,3 +108,34 @@ jobs:
       run: python -m pip install --upgrade tox codecov
     - name: Run tests
       run: tox ${{ matrix.toxargs }} -e ${{ matrix.toxenv }} -- ${{ matrix.toxposargs }}
+
+  parallel_and_32bit:
+    name: 32-bit and parallel
+    runs-on: ubuntu-latest
+    container:
+      image: quay.io/pypa/manylinux1_i686
+    steps:
+    - name: Checkout code
+      uses: actions/checkout@v2
```

Comment:
Looks like this doesn't work with container?
```
/usr/bin/docker exec  .... sh -c "cat /etc/*release | grep ^ID"
OCI runtime exec failed: exec failed: container_linux.go:370: starting container process caused: no such file or directory: unknown
```

Reply:
Looks like a known and unresolved issue at actions/checkout#334

Reply:
Subscribed!